docs/databricks-monitoring/databricks-monitoring.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Monitoring Azure Databricks in Azure Log Analytics
+titleSuffix: A library for monitoring Azure Databricks in Azure Monitor
+description: A scala library to enable monitoring of metrics and logging data in Azure Log Analytics
+author: petertaylor9999
+ms.date: 02/28/2019
+ms.topic:
+ms.service:
+ms.subservice:
+---
+
+# Monitoring Azure Databricks in Azure Log Analytics
+
+[Azure Databricks](/azure/azure-databricks/) is a fast, powerful, and collaborative [Apache Spark](https://spark.apache.org/)–based analytics service that makes it easy to rapidly develop and deploy big data analytics and artificial intelligence (AI) solutions. Most users take advantage of the simplicity of notebooks in their Azure Databricks solutions. For users that require more robust computing options, Azure Databricks supports the distributed execution of custom application code written in Scala.
+
+Monitoring in custom application code is a critical part of any production-level solution, and Azure Databricks offers robust functionality for monitoring custom application metrics, streaming query event information, and application log messages. Azure Databricks supports delivery of metrics and logging data to many different logging services, and the code library that accompanies this document extends the core monitoring functionality of Azure Databricks to send streaming query event information to [Azure Monitor](/azure/azure-monitor/overview).
```

Comment:
Change "streaming query event information" to something like "Spark metrics, events, and logging information" or something like that.